projects/plugins/jetpack/extensions/blocks/ai-assistant/extensions/multiple-blocks-edition/generate-panel.tsx

```diff
@@ -168,6 +170,10 @@ export default function GenerateContentPanel( { blocksIds } ) {
 		updateBlockAttributes,
 	] );
 
+	if ( ! connected ) {
+		return null;
+	}
```

Comment:
In my opinion, it would be best to exclude it from the [block filtering process](https://github.com/Automattic/jetpack/blob/20c344962cb6d2c131fb4d1f26b083494c00dbcb/projects/plugins/jetpack/extensions/blocks/ai-assistant/extensions/multiple-blocks-edition/index.ts#L12).